apis/Google.Cloud.Bigtable.Admin.V2/Google.Cloud.Bigtable.Admin.V2/BigtableTableAdminClientPartial.cs

```diff
@@ -0,0 +1,148 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax;
+using Google.Api.Gax.Grpc;
+using Google.Protobuf;
+using System.Threading.Tasks;
+
+namespace Google.Cloud.Bigtable.Admin.V2
+{
+    public partial class BigtableTableAdminClient
+    {
+        /// <summary>
+        /// Permanently drop/delete all rows in the table.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// This method simply delegates to <see cref="DropRowRangeAsync(DropRowRangeRequest, CallSettings)"/>.
+        /// </para>
+        /// </remarks>
+        /// <param name="tableName">
+        /// The unique name of the table on which to drop the rows. Must not be null.
+        /// </param>
+        /// <param name="callSettings">
+        /// If not null, applies overrides to this RPC call.
+        /// </param>
+        /// <returns>
+        /// A Task that completes when the RPC has completed.
+        /// </returns>
+        public Task DropAllRowsAsync(
```

Comment:
For most async methods (with flattenings) we have an overload accepting a cancellation token instead of a CallSettings (that just delegates to the CallSettings one). Could we add that here?

(Ditto DropRowRangeAsync)

Reply:
Done